TrackerEvents.cs

```diff
@@ -71,16 +42,22 @@ public static void UnregisterEventHandlers()
         /// <param name="obj">The current Dynamo workspace</param>
         private static void OnCurrentWorkspaceChanged(Dynamo.Graph.Workspaces.IWorkspaceModel obj)
         {
-            string FileName = obj.Name;
-            MessageBox.Show($"Congratulations on opening the {obj.Name} workspace!");
+            // When the Dynamo Workspace is changed, set the filename
+            ExportData.filename = obj.Name;
+
+            // Display a MessageBox to the user
+            // @todo Remove this as it's for testing purposes only
```

Comment:
instead of removing the `MessageBox`, use a [C# IF conditional compilation statement](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if). This way, when compiliing with `Debug` build setup, you see it, and when you compile with `Release` you don't.

Reply:
Thanks for this @radumg it's exactly what we need. I am however running into an issue with it. Perhaps you can advise? See issue: https://github.com/teamtreedyn/Binoculars/issues/22

Reply:
Marking this resolved as #if DEBUG added in https://github.com/teamtreedyn/Binoculars/commit/9046734bf8ef5ace6a5dc3a943674d0975750ec1